my-todo-list/src/app/shared-components/todo-add/todo-add.component.scss

```diff
@@ -0,0 +1,13 @@
+.btn-circle {
+    border-radius: 50%;
+    text-align: center;
+    float: right;
+    margin-right: 2.5%;
+}
+
+.addBtn {
+    font-size: 200%;
+    position: flex;
```

Comment:
Nu exista position flex. Flex este o proprietate doar pentru display. Position poate fi static, relative, absolute, etc. Poti incerca sa pozitionezi absolut butonul, fata de un parinte, caruia sa-i dai position relative. Un element absolute se pozitioneaza in functie de primul parinte pozitionat relative.